codes/quantum/qubits/quantum_expander.yml

```diff
@@ -0,0 +1,118 @@
+  # accepts a list of text entries, with one item per piece of information.
+  #
+  # Any fields below should be omitted if they do not apply.
+
+  rate: 'constant'
+
+  # This field that starts a list of strings, indexed by a dash; do not leave it
+  # empty. If empty, comment out.
+  # encoders: 
+  #   - 'Unitary circuit of depth ... \cite{arxiv:old-paper}.'
+  #   - 'Measurement-based preparation ... with ancilla overhead of ... .'
+  #   - 'Leave discussion of fault tolerance to fault-tolerance field.'
+
+  # transversal_gates: 'Transversal ... gates \cite{doi:ok-paper}. Comment out if doesn''t apply.'
+
+  # general_gates:
+  #   - 'Universal gate set achieved by either additional ... gate.'
+  #   - 'Magic-state distillation protocols'
+  #   - 'kth Clifford hierarchy gates obtained by ... circuits'
+
+  decoders: 
+    - '\cite{doi:10.1109/FOCS.2015.55} detail a linear time decoder, which corrects \(\Omega(n^{1/2})\) adversarial errors.'
+    # - 'Just-in-time decoder with ... \cite{arxiv:awesome-paper}.'
+
+  # fault_tolerance:
+  #   - 'Transversal gates are fault-tolerant w.r.t. ... noise \cite{doi:ok-paper}'
+  #   - 'Other fault-tolerant gadgets (measurements, encoders, error correcting steps)'
+  #   - 'Noise-model-preserving gadgets, noise-biased gates, fault-tolerant flag error correction'
+  #   - 'Pieceable fault tolerance.'
+
+  threshold: 
+    - 'Current estimate of \(2.7 \cdot 10^{-16}\) in locally stochastic noise model \cite{arXiv:1711.08351}.'
+
+  
+  # Propose a field that you feel is applicable to a large and interesting class of codes
+  #some_other_field:
+  # - 'detail'
+
+# realizations:
+#   # List and explain the different "domains" of realizations in list items.
+#   # Group closely related realizations into a single item. It's OK to have a
+#   # single list item. Each item should more or less correspond to what would be
+#   # said in a paragraph.
+#   - 'Not directly implemented, but BCH codes form a subclass, and are used in DVD, solid state drive storage, etc.'
+
+notes:
+  - 'Distance typically scales like \(\Omega(n^{1/2})\).'
+  # - 'Bounds on n,k, or d for this class.'
+  # - 'Links to code tables, github, GAP algebra packages, more papers \cite{arXiv:####.#####}.'
+  # - 'Connections to physics; based on \(H^4(\mathcal{BG},\mathbb{Z})\)'
+  # - 'Can reference other codes, but try to put such connections in the relations fields.'
+  # - 'Anything noteworthy that''s not above or below.'
+
+relations:
+  parents:
+    - code_id: css
+    - code_id: quantum_ldpc
+    #   detail: 'Proven \cite{doi:10.1504/IJICOT.2011.044674}'
+    # - code_id: code_id2
+    #   detail: 'code_id2 is a code family that is constructed in a seemingly different way than this code, yet still includes this code.'
+    # - code_id: code_id3
+    #   detail: 'code_id3 is code family described by a property shared by this code other than physical space (e.g., topological)'
+
+  cousins:
+    - code_id: ldpc
```

Comment:
LDPC and QLDPC are already cousins, so connection is made upstairs. Should justify why a connection here is necessary; is there a feature of these particular codes that is more intimately related to LDPC codes than a generic QLDPC code?